scripts/index.js

```diff
@@ -1,25 +1,41 @@
+// worked with Eric M.
 
 // Implement the following functions to fulfill the tests!
-function getLocationName() {}
-function getLocationCountry() {}
-function getLocationLatitude() {}
-function getLocationLongitude() {}
-function getDescription() {}
-function getWindSpeed() {}
-function getSunrise() {}
+function getLocationName(weather) {
+  const city = new Map();
+  const cityValues = Object.values(weather); // array of city values
+  return cityValues[10]; //city name is in index 10
+}
 
+function getLocationCountry(weather) {
+  console.log(weather);
+  return weather.sys.country;
+}
+function getLocationLatitude(weather) {
+  return weather.coord.lat;
+}
+function getLocationLongitude(weather) {
+  return weather.coord.lon;
+}
+function getDescription(weather) {
+  return weather.weather[0].description;
+}
+function getWindSpeed(weather) {
+  return weather.wind["speed"];
```

Comment:
Excellent! Would it still return the correct value if you used `.speed` instead of `["speed"]`?